SRC/NETWORK/hget.c

```diff
@@ -123,32 +135,32 @@ enum TcpipErrorCodes {
 #define strDefaultFilename "index.htm";
 
 const char* strTitle=
-    "HTTP file downloader 1.1\r\n"
-    "By Konamiman, 1/2011\r\n"
+    "HTTP file downloader 1.3\r\n"
+    "By Konamiman, 2011 v1.3 Oduvaldo (ducasp@gmail.com) 07/2019\r\n"
     "\r\n";
-    
+
 const char* strUsage=
-    "Usage: hget <file URL>|<local file with the URL>|con [/l:<local file name>]\r\n"
-    "            [/c] [/v] [/h] [/x:<headers file name>] [/a:<user>:<password>]\r\n"
+    "Usage: hgetf <file URL>|<local file with the URL>|con [/l:<local file name>]\r\n"

```

Comment:
Why `hgetf`?

Reply:
Forgot to remove it from my previous customized version :)